aomi/model/auth.py

```diff
@@ -86,6 +97,18 @@ class AppRole(Auth):
     required_fields = ['name', 'policies']
     config_key = 'approles'
 
+    @staticmethod
```

Comment:
https://stackoverflow.com/questions/38987/how-to-merge-two-python-dictionaries-in-a-single-expression#answer-26853961